base/doc/ltnews36.tex

```diff
@@ -0,0 +1,152 @@
+% \iffalse meta-comment
+%
```

Comment:
I thought there is already a ltnews36 ... might be that it is in the umerged branch made by Ulrike (which should get merged)

Reply:
not in devlop, and I wanted to add a para somewhere. We should be able to handle that when merging the other branch (unless that is ready to go in first, in which case I'll rebase this after that goes in)

Reply:
Ulrike's PR now merged, touched exactly same set of files as this, cleaning up merge conflicts here...